domain/schema/controller/sql/0017-user-permission.sql

```diff
@@ -125,4 +125,5 @@ SELECT
     p.access_type,
     p.object_type
 FROM v_permission AS p
-WHERE p.grant_on = 'controller' AND p.object_type = 'controller';
+INNER JOIN controller ON p.grant_on = controller.uuid
```

Comment:
Update the view comment to `-- All controller permissions, verifying the controller does exist.` Matches the others.

Reply:
Changed.